oracle/key.pem

```diff
@@ -0,0 +1,8 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
❓ Just to make sure my understanding is correct - this is the private key for the local regtest default oracle, correct?

Reply:
Yes, this is only for the local setup.